src/commands/exportComments.ts

```diff
@@ -62,5 +48,29 @@ export async function exportVersionComments(uri: vscode.Uri) {
   }
 
   const comments = await compileComments(guid, version);
-  await exportComments(comments);
+  await exportComments(comments, uri);
+}
+
+export async function exportComments(
+  compiledComments: string,
+  uri: vscode.Uri
+) {
+  const deleteCachedComments = await getDeleteCommentsPreference();
+
+  const document = await vscode.workspace.openTextDocument({
+    content: compiledComments,
+    language: "text",
+  });
+
+  vscode.window.showTextDocument(document, vscode.ViewColumn.Beside);
+
+  if (compiledComments) {
+    vscode.env.clipboard.writeText(compiledComments);
+    vscode.window.showInformationMessage("Comments copied to clipboard.");
+  }
+
+  if (deleteCachedComments) {
```

Comment:
Nit: If the variable isn't going to be used until later, it's best to move it's declaration closer to it's use location. This helps with readability as the reader doesn't have to hold as much state in mind while reading through the code. It also has the (generally very minor) benefit that an early return will not perform unnecessary work.
```suggestion
  const document = await vscode.workspace.openTextDocument({
    content: compiledComments,
    language: "text",
  });

  vscode.window.showTextDocument(document, vscode.ViewColumn.Beside);

  if (compiledComments) {
    vscode.env.clipboard.writeText(compiledComments);
    vscode.window.showInformationMessage("Comments copied to clipboard.");
  }

  const deleteCachedComments = await getDeleteCommentsPreference();
  if (deleteCachedComments) {
```